ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -161,48 +181,65 @@
     - name: Install Node JS
       ansible.windows.win_command: "nvm install {{ node_version }}"
 
-    - name: Run NPM version
-      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm -v"
-      register: npm_version_out
+    - name: Copy test suite
+      ansible.builtin.copy:
+        src: ../../integration-tests/test-suite
+        dest: 'C:\Windows\Temp\'
+
+    - name: Install test-suite dependencies
+      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm i"
+      args:
+        chdir: '{{ test_suite_folder }}'
 
-    - name: NPM version result (to be removed)
-      debug:
-        var: npm_version_out
+    - name: Remove previous reports
+      ansible.windows.win_file:
+        path: '{{ test_suite_folder }}\reports'
+        state: absent
 
-    - name: Run Node version
-      ansible.windows.win_shell: "nvm use {{ node_version }} ; node -v"
-      register: node_version_out
+    - name: Run test-suite
+      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm run test"
+      args:
+        chdir: '{{ test_suite_folder }}'
+        creates: '{{ test_suite_folder }}\reports'
+      environment:
+        LOGGING_ENDPOINT: https://staging-log-api.newrelic.com/log/v1
+        NERD_GRAPH_URL: https://staging-api.newrelic.com/graphql
+        API_KEY: "{{ new_relic_api_key }}"
+        ACCOUNT_ID: "{{ new_relic_account_id }}"
+        MONITORED_FILE: '{{ monitored_file }}'
+#        MONITORED_TCP_PORT: "{{ monitored_tcp_port }}" # waiting for https://github.com/fluent/fluent-bit/issues/6671
+        MONITORED_WINDOWS_LOG_NAME_USING_WINLOG: "{{ monitored_windows_log_name_using_winevtlog }}"
+        MONITORED_WINDOWS_LOG_NAME_USING_WINEVTLOG: "{{ monitored_windows_log_name_using_winlog }}"
 
-    - name: Node version result (to be removed)
-      debug:
-        var: node_version_out
+    - name: Fetch results
+      fetch:
+        flat: true # behave like copy, we only want the file, not the whole path
+        src: '{{ test_suite_report_path }}'
+        dest: '{{ test_reports_dir }}/{{ fb_version }}_{{ os_distro }}_{{ os_version }}_{{ arch }}.xml'
 
 - name: Process and merge test reports
   hosts: localhost
-  become: true
   vars:
     node_version: 18.18.0
+    controller_scripts_folder: ../../integration-tests/controller-scripts
```

Comment:
Hmmmmmmmm isn't this just `~/fluent-bit-package/integration-tests/controller-scripts`? We could modify the `make local` target a little bit so that we just get a copy of the original repo into `fluent-bit-package` instead of `fluent-bit-package-copy` so it resembles what gets executed in the Fargate task and aligns with this.